packages/story-editor/src/components/library/panes/text/textPane.js

```diff
@@ -64,6 +90,11 @@ function TextPane(props) {
     []
   );
 
+  const handleToggleClick = useCallback(() => {
+    setUseSmartColor((currentValue) => !currentValue);
+  }, [setUseSmartColor]);
```

Comment:
Nit:
```suggestion
  const handleToggleClick = useCallback(
    () => setUseSmartColor((currentValue) => !currentValue),
    [setUseSmartColor]
  );
```

I actually think we do this `setFoo((val) => !val)` a lot. We could create a very simple hook for that, e.g. `const handleToggleClick = useToggle(setUseSmartColor)`. It might be overkill, but I like simplifying things.